package.json

```diff
@@ -43,29 +43,16 @@
     "upupdowndownleftrightleftrightbastart"
   ],
   "dependencies": {
-    "@babel/core": "^7.20.12",
-    "@babel/eslint-plugin": "^7.19.1",
-    "@babel/plugin-proposal-decorators": "^7.20.7",
-    "@babel/plugin-proposal-export-default-from": "^7.18.10",
-    "@babel/plugin-transform-react-inline-elements": "^7.18.6",
-    "@babel/plugin-transform-runtime": "^7.19.6",
-    "@babel/preset-env": "^7.20.2",
-    "@babel/preset-react": "^7.18.6",
-    "@babel/preset-typescript": "^7.18.6",
-    "@babel/runtime": "^7.20.7",
-    "@enact/dev-utils": "^5.1.0",
+    "@enact/dev-utils": "github:enactjs/dev-utils#feature/WRO-697",
     "@enact/template-sandstone": "^1.5.2",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.10",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
-    "@typescript-eslint/eslint-plugin": "^5.48.2",
```

Comment:
why this module has been removed?

Reply:
I misunderstood unify scope. I will restore and will investigate this module in WRP-6904